vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -136,10 +136,38 @@
   register: arkcase_war
   when: arkcase_retrieve_strategy == "sftp"
 
+- name: download arkcase war file (maven)
+  become: yes
+  become_user: arkcase
+  maven_artifact:
+    repository_url: "{{ repository_url }}/{{ arkcase_artifact_id }}.{{ arkcase_artifact_repository_type }}/"
+    username: "{{ nexus_username }}"
```

Comment:
Do we need to specify a user and password?  The core ArkCase and config server artifacts can be public.

Reply:
Nice catch. I did not thought of that, and need to investigate. Thanks!

Reply:
Core component, config-server, configuration and arkcase war can be downloaded using maven retrieve strategy without username or password. Using the code as it is, no need to change anything.